src/combinators/mod.rs

```diff
@@ -1,6 +1,6 @@
-use gen::GenError;
-use lib::std::io::Write;
-use lib::std::io;
+use crate::gen::GenError;
+use crate::lib::std::io::Write;
+use crate::lib::std::io;
```

Comment:
maybe make this `use crate::lib::std::io::{self, Write}` ?

Reply:
Done